k8s/coordinator/development/config.toml

```diff
@@ -1,18 +1,20 @@
 [log]
-filter = "xaynet=debug,info"
+filter = "xaynet=debug,http=warn,info"
 
 [api]
 bind_address = "0.0.0.0:8081"
 
 [pet]
-min_sum = 1
-min_update = 3
-sum = 0.01
-update = 0.1
+min_sum_count = 1
+min_update_count = 3
+min_sum_time = 5
+min_update_time = 10
+sum = 0.5
+update = 0.9
```

Comment:
Were these changed on purpose? Seems unrelated to the rest of the PR

Reply:
Well, I caught these changes from the default config.toml at configs/config.toml which were already merged to master.

Reply:
I am willing to submit these changes/updates to the `config.toml` that lives in the cluster on a separate PR, just so it doesn't get mixed with the rest of the work done on this branch.

Reply:
Doing it in this PR is fine for me, I was just making sure it didn't slip in by accident ;)